### PR TITLE
Torch es

### PR DESCRIPTION
**Pull request by wangzelong0663**
Upgrade the interface of the ES of torch version.

### CODE REVIEW

File: benchmark/torch/ES/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Based on PARL, we have implemented the Evolution Strategies (ES) algorithm and evaluate it in Mujoco environments. Its performance reaches the same level of indicators as the paper.
+
++ ES in
+[Evolution Strategies as a Scalable Alternative to Reinforcement Learning](https://arxiv.org/abs/1703.03864)
+
+### Mujoco games introduction
+Please see [here](https://github.com/openai/mujoco-py) to know more about Mujoco games.
+
+### Benchmark result
+![learninng_curve](result_es.png)
```
Comment (zenghsh3) [inline benchmark/torch/ES/README.md]: Use the consistent drawing tools and use the training steps as the x-axis.

**Review by zenghsh3 — CHANGES_REQUESTED** [review]

**Review by zenghsh3 — DISMISSED** [review]
LGTM

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
